wiki/LocalSettings.php

```diff
@@ -172,9 +172,38 @@
 # End of automatically generated settings.
 # Add more configuration options below.
 wfLoadExtension( 'MobileFrontend' );
-
+wfLoadExtension( 'PluggableAuth' );
+wfLoadExtension( 'SimpleSAMLphp' );
 wfLoadSkin( 'MinervaNeue' );
 $wgMFDefaultSkinClass = 'SkinMinerva';
 
 # MORE DEBUGGING
 $wgDebugLogFile = "/var/log/mediawiki/debug-{$wgDBname}.log";
+
+# Configure user rights
+$wgGroupPermissions['*']['edit'] = false;
+$wgGroupPermissions['user']['edit'] = true;
+$wgGroupPermissions['*']['createaccount'] = true;
+$wgGroupPermissions['*']['autocreateaccount'] = true;
+
+# Config simplesamlphp
+
+$wgSimpleSAMLphp_InstallDir = 'simplesamlphp';
+$wgSimpleSAMLphp_AuthSourceId = 'default-sp';
+$wgSimpleSAMLphp_UsernameAttribute = [
+	'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname',
+	'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname'
+];
+$wgSimpleSAMLphp_EmailAttribute = 'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name';
+$wgSimpleSAMLphp_RealNameAttribute = [
```

Comment:
I think we can use http://schemas.microsoft.com/identity/claims/displayname here instead, but it should normally be the same as given+surname anyway

Reply:
plus the xmlsoap.org ones look more standardised, so there's good reason to use them anyway